integrationTests/testInitializer.go

```diff
@@ -169,15 +173,19 @@ func CreateMetaStore(coordinator sharding.Coordinator) dataRetriever.StorageServ
 }
 
 // CreateAccountsDB creates an account state with a valid trie implementation but with a memory storage
-func CreateAccountsDB(shardCoordinator sharding.Coordinator) *state.AccountsDB {
-	hasher := sha256.Sha256{}
-	store := CreateMemUnit()
+func CreateAccountsDB(shardCoordinator sharding.Coordinator,
```

Comment:
there's no need to split the function signature in two lines

Reply:
Changed back to one line.